src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -18,23 +18,71 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       show: false,
     },
     tooltip: {
-      format: {
-        title: (date) => {
-          return intl.formatDate(date, {
+      contents: function(data) {
+        if (data.length) {
+          const date = intl.formatDate(data[0].x, {
             timeZone: 'UTC',
             day: 'numeric',
             month: 'long',
             year: 'numeric',
             weekday: 'long',
           });
-        },
-        value: (value) => {
-          return (
-            intl.formatMessage({
-              id: 'reports_daily_visits.tooltip_page_views',
-            }) + value
-          );
-        },
+          let quantity = '',
+            withEmail = '',
+            withoutEmail = '';
+          for (const item of data) {
+            switch (item.id) {
+              case 'quantity':
+                quantity =
+                  intl.formatMessage({
+                    id: 'reports_daily_visits.tooltip_page_views',
+                  }) + item.value;
+                break;
+              case 'withEmail':
+                withEmail =
+                  item.value !== null
+                    ? intl.formatMessage({
+                        id: 'reports_daily_visits.tooltip_with_email',
+                      }) + item.value
+                    : '';
+                break;
+              case 'withoutEmail':
+                withoutEmail =
+                  item.value !== null
+                    ? intl.formatMessage({
+                        id: 'reports_daily_visits.tooltip_without_email',
+                      }) + item.value
+                    : '';
+                break;
+              default:
+                return '';
+            }
+          }
+          return `
+          <table class="c3-tooltip">
+            <tbody>
+              <tr>
+                <th colspan="2">${date}</th>
+              </tr>
+              <tr>
+                <td colspan="2">
+                ${withEmail === '' ? quantity : ''}
```

Comment:
What do you think about

```suggestion
                ${withEmail || quantity}
```

Reply:
Uh, no estoy seguro de si esta opción es correcta, ya que va a usar _quantity_ incluso cuando _withEmail_ es cero, y probablemente eso esté bien.

Reply:
Condition validation changed.